src/leon3_serial_ccsds/leon3_serial_ccsds.c

```diff
@@ -0,0 +1,265 @@
+/**@file
+ * This file is part of the Leon3 Serial CCSDS Driver for the Test Environment.
```

Comment:
This is not test environment -> please change to TASTE Leon3 Runtime

Reply:
ack